src/include/rho/FixedVector.hpp

```diff
@@ -222,9 +222,8 @@ namespace rho {
 	// Virtual function of GCNode:
 	void detachReferents() override;
     private:
-	T* m_data;  // pointer to the vector's data block.
```

Comment:
The most interesting case for putting the data elsewhere is doing lazy evaluation of large vector calculations.  By batching up multiple vector ops and doing a single pass through the data, you can get big performance gains.  To make that robust against user code written against the C API, you want to store the data in an inaccessible memory page, so you can detect code that's trying to use the data, and compute it just in time.

Sharing the location for m_data and m_first_element_storage might work.  We'd need to steal a bit from somewhere to tell which case we have, and we'd have to check that bit a lot, so it'd need to be benchmarked.